src/OrganisationUnitTree/useForceReload.js

```diff
@@ -0,0 +1,21 @@
+import { useEffect, useState } from 'react'
+
+/**
+ * This will create a new reloadId everytime "forceReload" changes to true,
+ * which can be used as the "key" prop on the org unit tree.
+ * When that id changes, the whole tree rerenders
+ * and therefore triggers all "useDataQuery"s to
+ * run the query again
+ *
+ * @param {bool} forceReload
+ * @returns {Int}
+ */
+export const useForceReload = forceReload => {
+    const [reloadId, setReloadId] = useState(0)
+
+    useEffect(() => {
+        forceReload === true && setReloadId(reloadId + 1)
+    }, [forceReload])
+
+    return reloadId
+}
```

Comment:
So if I understand this correctly:
- we sometimes need to force reloading of certain nodes
- to achieve this we are currently updating the component's `key` to trigger a rerender
- this in turn will trigger the component to fetch again

Perhaps this works fine, but I am slightly concerned about this approach because I find this quite an indirect solution to the problem... I can't fully oversee the potential problems, but I can imagine there might be many. I wonder how this would behave when:
- A single node needs several reloads, will the current helper ever return a value other than `1` or `0`?
- An unrelated prop changes, after a forced reload. I think this would trigger a yet another reload.

Reply:
The number will increment above 1. And it won't trigger a second reload as the dependency array acts as a guard to only execute this once when the `forceReload` value changes

Reply:
👍 Thanks for clarifying, this logic seems completely sound then. This comment was also connected to my general remark reg. meta-state, and since we decided to cancel that for now, I think what you have now is great..